File: katas/8 kyu/string-repeat/solution.py
# string-repeat
import codewars_test as test
# -------------------------------- MY SOLUTION ------------------------------- #
repeat_str = lambda a,b : a*b
# ----------------------------------- TEST ----------------------------------- #
@test.describe('Fixed tests')
def basic_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(repeat_str(4, 'a'), 'aaaa')
        test.assert_equals(repeat_str(3, 'hello '), 'hello hello hello ')
        test.assert_equals(repeat_str(2, 'abc'), 'abcabc')

File: katas/8 kyu/string-repeat/solution.js
// string-repeat
/* ------------------------------- MY SOLUTION ------------------------------ */
function repeatStr (n, s) {
  return s.repeat(n)
}
/* ----------------------------- CLEVER SOLUTION ---------------------------- */
/* ---------------------------------- TEST ---------------------------------- */
const chai = require("chai");
const Test = chai.assert.strictEqual; // replace Test.assertEqual with Test
chai.config.truncateThreshold=0;

describe("Tests", function() {
  it ("Basic tests", function() {
    Test(repeatStr(3, "*"), "***");
    Test(repeatStr(5, "#"), "#####");
    Test(repeatStr(2, "ha "), "ha ha ");
  });
});